client/src/charts/wrapped_nivo/nivo_line.js

```diff
@@ -78,46 +78,51 @@ export class NivoResponsiveLine extends React.Component {
       isInteractive,
       motionDamping,
       motionStiffness,
+      custom_table,
     } = this.props;
 
     const {
       y_scale_zoomed,
     } = this.state;
     
-    const table_data = _.chain(data)
-      .map(row => {
-        const series_name = row.id;
-        return _.chain(row.data)
-          .map(series => (_.isNil(series.y) ? undefined : {label: series.x, [series_name]: series.y}) )
-          .value();
-      })
-      .flatten()
-      .compact()
-      .groupBy('label')
-      .map( _.spread(_.merge) )
-      .value();
-    const line_table_value_formatter = (value) =>
+    const get_table_nivo_line = () => {
```

Comment:
The `_nivo_line` part can be inferred from the file it's in? Just being pedantic, but `get_table_nivo_line` just felt a little awkward.